lib/ansible/modules/database/mysql/mysql_variables.py

```diff
@@ -177,7 +214,14 @@ def main():
         else:
             module.fail_json(msg="unable to find %s. Exception message: %s" % (config_file, to_native(e)))
 
-    mysqlvar_val = getvariable(cursor, mysqlvar)
+    mysqlvar_val = None
+    if mode == 'global' or mode == 'persist':
```

Comment:
Why not
```suggestion
    if mode in ('global', 'persist'):
```
? :)

Reply:
you won't believe it was initially like you're suggesting :) when i was debugging i rewrote this part like it is now :)
I made a real sin - wrote code just before to go to bed... :)

Reply:
maybe it is because i'm trying to repeat C++ at this time :)